swap/prices.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Swarm Authors
+// This file is part of the Swarm library.
+//
+// The Swarm library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Swarm library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Swarm library. If not, see <http://www.gnu.org/licenses/>.
+
+package swap
+
+/*
+This module contains the pricing for message types as constants.
+
+Pricing in Swarm is defined in an internal unit.
+The name of this internal unit is honey.
+The honey unit allows to set prices of messages relative to each other
+independently of any currency.
+
+The expectation is then that an external, probably on-chain, **oracle**
+would be queried with the total amount of honey for a message,
+for which the oracle would return the price in a given currency.
+
+Currently the expected currency from the oracle would be wei,
+but it could potentially be any currency the oracle and Swarm support,
+allowing for a multi-currency design.
+*/
+
+// Placeholder prices
```

Comment:
remove comment

Reply:
i disagree here; i don't think it's immediately obvious that these are placeholder prices, even while having a value of `1`.

Reply:
ok... can you please explain what do you mean by "placeholder prices"? i don't seem to get it

Reply:
a reasonably justified value for prices will be arrived at after experimentation. as of now, we don't really have much of an idea of what numbers to use as price for each type of message–this is what we mean by `placeholder`.